test/TestSetup.sol

```diff
@@ -118,21 +117,24 @@ contract TestSetup is Test, Configured {
         avatar.enableModule(module);
     }
 
+    function _executeTestTransaction(string memory filename) internal {
+        // This is so we can just call execTransactionFromModule to simulate executing transactions without signatures.
+        _addModule(IAvatar(morphoDao), address(this));
+        _addModule(IAvatar(operator), address(this));
+
+        Transaction memory transaction = _getTxData(filename);
+
+        morphoDao.execTransactionFromModule(address(delayModifier), 0, _wrapTxData(transaction), Operation.Call);
```

Comment:
I'm thinking that we should maybe consider having the transaction wrapped by default in the raw jsons. This is because we would like for the transaction parameters in the json to be exactly the same as what the safe transaction should be. The wrap tx data could still be useful for actually forming and verifying these transactions in solidity though.

Reply:
It depends of who will use the repository because when you verified the transaction on Safe, it is simpler to add the unwrapped transaction. I think it is a question of usage. What do you think, which usage is the most important?

Reply:
I think we can merge this PR. You could build yours on top.